docs/wintercredit/lexicon.rst

```diff
@@ -0,0 +1,84 @@
+Lexicon
+=======
+
+.. toctree::
+   :maxdepth: 4
+
+The following are the terms in use in this module and coming as much as
+possible from the official Hydro-Quebec electricy rates document (page
+31) :
+
+https://www.hydroquebec.com/data/documents-donnees/pdf/electricity-rates.pdf
+
+and from the “Régie de l’énergie” document :
+
+http://publicsde.regie-energie.qc.ca/projets/469/DocPrj/R-4057-2018-B-0062-DDR-RepDDR-2018_10_26.pdf#page=124
+
+**Period**
+    A period is time window where a specific billing or algorythmic logic is
+    applied by HQ
+
+        * normal
+            A period when nothing special is defined by HQ rate policies
+
+        * peak / peak hours
+            In this module:
+                All hours from 06:00 to 09:00 and from 16:00 to 20:00 during the winter.
+
+            This is when the critical peak events from hydro are happening.
+
+            In hydro’s document there are also exclusions for specific holiday dates (Christmans, New year, Good Friday
+            and Easter Monday) that we don’t take into account here (yet)
+
+        * anchor / temperature adjustment
```

Comment:
I would scrap the temperature adjustment from here if everyone agrees we should call it anchor. Just leave the note in the description that mentions it is referred to temperature adjustement in other docs.